.wiki/Dashboard.md

```diff
@@ -0,0 +1,70 @@
+# Artipie dashboard
+
+The Artipie provides [front-end](github.com/artipie/front) web application to manage repositories using a web-browser.<br/>
+The Artipie front-end provides a convenient dashboard with UI pages for managing repositories.<br/>
+Artipie front-end is distributed as [Docker image](hub.docker.com/r/artipie/front) and as fat jar.
+The jar file can be downloaded on GitHub [releases page](https://github.com/artipie/front/releases).
+
+The Artipie front-end is independent part of Artipie project that interacts with Artipie server by using REST-API services provided by Artipie server.
+
+The Artipie dashboard provides following abilities:
+* Sign in
+* View of all available repositories
+* Creating a new repository
+* Edit a repository
+* Remove a repository
+* Sign out
+
+To get more information about start up the Artipie front-end in Docker container please visit article ["Quickstart with docker-compose"](./DockerCompose) 
+ 
+## Sign in
+
+The `Sign in` page allows the Artipie users to login to Artipie dashboard.<br/> 
+Page has two fields, `User`, and the `Password`.<br/>
+The inputs will be verified by Artipie-server. 
+Once the verification complete, the user will proceed to the list of repositories.
+
+[[/images/dashboard/signin.jpg|Sign in]]
+
+## Repository list
+The `Repository list` page allows the Artipie user to view all available repositories.
+The page provides short information for each repository such as:
+* Repository name
+* Repository type
+* Repository port where Artipie-server serves http requests to repository from according tools
+
+[[/images/dashboard/repository_list.jpg|Repository list]]
+
+The Artipie user can filter repositories by a repository name:
+[[/images/dashboard/repository_list_filtered.jpg|Filtered repository list]]
+
+## Create repository
```

Comment:
I'd suggest to add link to [this](https://github.com/artipie/artipie/wiki/Configuration-Repository) wiki page in repository subsection and write smth like "Follow this link to know all the technical details about supported repositories and settings".